services/webnn/ort/graph_builder_ort.cc

```diff
@@ -245,6 +246,46 @@ std::string GraphBuilderOrt::GenerateNextOperationName(std::string_view label) {
                           kUnderscore);
 }
 
+const std::vector<const char*> GetRecurrentNetworkActivations(
+    std::vector<mojom::RecurrentNetworkActivation> activations,
+    uint32_t num_directions) {
+  std::vector<const char*> activation_list;
+  for (const auto& activation : activations) {
+    switch (activation) {
+      case mojom::RecurrentNetworkActivation::kRelu:
+        activation_list.push_back("relu");
+        break;
+      case mojom::RecurrentNetworkActivation::kSigmoid:
+        activation_list.push_back("sigmoid");
+        break;
+      case mojom::RecurrentNetworkActivation::kTanh:
+        activation_list.push_back("tanh");
+        break;
+      default:
+        NOTREACHED() << "Unsupported recurrent network activation function.";
+    }
+  }
+  if (num_directions == 2) {
```

Comment:
There are no other num_directions to handle. Just use a bool parameter `is_bidirectional`.